webpack/test-utils/react-testing-lib-wrapper.js

```diff
@@ -0,0 +1,39 @@
+// For react-testing-library helpers, overrides, and utilities
+// See https://testing-library.com/docs/react-testing-library/setup for more info
+import React from 'react';
+import thunk from 'redux-thunk';
+import Immutable from 'seamless-immutable';
+import { render } from '@testing-library/react';
+import { createStore, applyMiddleware, combineReducers } from 'redux';
+import { Provider } from 'react-redux';
+import { Router } from 'react-router-dom';
+import { createMemoryHistory } from 'history';
+
+// Renders testable component with redux and react-router according to our usage
+function katelloRender(
```

Comment:
Took me a minute but I think you import `render` for regular testing and `katelloRender` when you need the Redux store attached.  So I think it would help if this were named more obviously, like `renderWithStore` or `renderWithKatelloRedux` or something.

Reply:
I like `renderWithKatelloRedux` since it uses the Katello namespacing, so it couldn't be used for other plugins or Foreman without changing to be more generic. I'll maybe add some more comments to this file to make things more clear on how this wrapper works and why it's needed.